src/common/common.py

```
@@ -199,21 +209,27 @@ def page_setup(page: str = "") -> dict[str, Any]:
+                
         else:
             st.session_state.workspace = Path(workspaces_dir, "default")
-            st.query_params.workspace = "default"
 
         if st.session_state.location != "online":
             # not any captcha so, controllo should be true
             st.session_state["controllo"] = True
 
-    if "workspace" not in st.query_params:
+    # If no workspace is specified and workspace feature is enabled, set default workspace and query param
+    if "workspace" not in st.query_params and st.session_state.settings["is_workspace_enabled"] == True:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if st.session_state.settings["is_workspace_enabled"]:` for truth checks

Replace with `st.session_state.settings["is_workspace_enabled"]`

(E712)